apps/web/src/components/Router.tsx

```diff
@@ -8,1039 +9,948 @@ import { defaultLogger } from "@gc-digital-talent/logger";
-                          import("../pages/Pools/EditPoolPage/EditPoolPage"),
-                      },
-                      {
-                        path: "pool-candidates",
-                        children: [
-                          {
-                            index: true,
-                            lazy: () =>
-                              import(
-                                "../pages/PoolCandidates/IndexPoolCandidatePage/IndexPoolCandidatePage"
-                              ),
-                          },
-                        ],
-                      },
-                      {
-                        path: "screening",
-                        lazy: () =>
-                          import(
-                            "../pages/Pools/ScreeningAndEvaluationPage/ScreeningAndEvaluationPage"
-                          ),
-                      },
-                      {
-                        path: "manage-access",
-                        lazy: () =>
-                          import(
-                            "../pages/Pools/ManageAccessPage/ManageAccessPage"
-                          ),
-                      },
-                      {
-                        path: "plan",
-                        lazy: () =>
-                          import(
-                            "../pages/Pools/AssessmentPlanBuilderPage/AssessmentPlanBuilderPage"
-                          ),
-                      },
-                    ],
-                  },
-                ],
-              },
-              {
-                path: "pools/:poolId/preview",
-                lazy: () =>
-                  import(
-                    "../pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage"
-                  ),
-              },
-              {
-                path: "pool-candidates",
-                lazy: () =>
-                  import(
-                    "../pages/PoolCandidates/AllPoolCandidatesPage/AllPoolCandidatesPage"
-                  ),
-              },
-              {
-                path: "candidates/:poolCandidateId/application",
-                lazy: () =>
-                  import(
-                    "../pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage"
-                  ),
-              },
-              {
-                path: "talent-requests",
-                children: [
-                  {
-                    index: true,
-                    lazy: () =>
-                      import(
-                        "../pages/SearchRequests/IndexSearchRequestPage/IndexSearchRequestPage"
-                      ),
-                  },
-                  {
-                    path: ":searchRequestId",
-                    lazy: () =>
-                      import(
-                        "../pages/SearchRequests/ViewSearchRequestPage/ViewSearchRequestPage"
-                      ),
-                  },
-                ],
-              },
-              {
```

Comment:
Did you mean to zap these?

Reply:
Oops, you zapped "../pages/Communities/UpdateCommunityPage/UpdateCommunityPage", too.

Reply:
:sob: added back in [re-add community update page](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12076/commits/69966aebcf62f558033c2b95aad3e2645ea5f319)